mkt/games/views.py

```diff
@@ -0,0 +1,23 @@
+from mkt.fireplace.serializers import (FireplaceESAppSerializer,
+                                       FireplaceESWebsiteSerializer)
+from mkt.games.filters import DailyGamesFilter
+from mkt.games.paginator import ESGameAggregationPaginator
+from mkt.search.serializers import DynamicSearchSerializer
+from mkt.search.views import MultiSearchView
+
+
+class DailyGamesView(MultiSearchView):
+    authentication_classes = []
+    cors_allowed_methods = ['get']
+    filter_backends = [DailyGamesFilter]
```

Comment:
Maybe, yeah. It's all curated, though PublicAppsFilter is good since apps can change without curators knowing. DeviceTpeFilter, maybe, curators are supposed to choose desktop only, but it probably wouldn't hurt.